src/android/TP6m.jsx

```diff
@@ -106,16 +109,19 @@ class TP6M extends React.Component {
 
   async componentDidUpdate(prevProps) {
     if (
-      this.props.test === prevProps.test &&
-      this.props.platform === prevProps.platform
-    )
+      ['test', 'platform', 'past', 'ending'].every(
+        v => this.props[v] === prevProps[v]
+      )
+    ) {
```

Comment:
With more properties (`past` and `ending`) that can be changed, we much check them to determine if we reload